packages/back-end/src/events/handlers/webhooks/webHooksEventHandler.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  NotificationEventName,
+  NotificationEventPayload,
+  NotificationEventResource,
+} from "../../base-types";
+import { WithOrganizationId } from "../../base-events";
+
+/**
+ * Common handler that looks up the web hooks and makes a post request with the event.
+ * @param payload
+ */
+export const webHooksEventHandler = async (
```

Comment:
This code is not yet used in this set of changes but will be used in the next set of changes.